team-validator.js

```diff
@@ -202,6 +178,19 @@ if (!process.send) {
 	});
 }
 
+/**
+ * Validates a username or Pokemon nickname
+ */
+
+var bannedNameStartChars = {'~':1, '&':1, '@':1, '%':1, '+':1, '-':1, '!':1, '?':1, '#':1, ' ':1};
+function getName (name) {
+	name = Tools.getName(name);
+	while (bannedNameStartChars[name.charAt(0)]) {
```

Comment:
Looks like https://github.com/Zarel/Pokemon-Showdown/commit/70ea1533a49198aa1c157e4d30897283182161b1 wasn't applied to team validator. Will do it. And we are left with just `Tools.getName``!